interceptor/opencensus/opencensus.go

```diff
@@ -100,13 +98,35 @@ func (oci *OCInterceptor) Export(tes agenttracepb.TraceService_ExportServer) err
 	traceBundler.DelayThreshold = spanBufferPeriod
 	traceBundler.BundleCountThreshold = spanBufferCount
 
+	var lastNonNilNode *commonpb.Node = initiatingNode
+
 	for {
 		recv, err := tes.Recv()
 		if err != nil {
 			return err
 		}
 
-		// Otherwise add these spans to the bundler
-		traceBundler.Add(recv.Spans, len(recv.Spans))
+		// If a Node hasn't been sent from downstream, use the last received node.
+		node := recv.Node
+		if node == nil {
```

Comment:
@bogdandrutu @songy23 how does this look?

Reply:
LGTM

Reply:
I would write it a bit different:
```go
if recv.Node != nil {
  lastNonNilNode = recv.Node
}
// Otherwise add them to the bundler.
bundlerPayload := &spansAndNode{node: lastNonNilNode, spans: recv.Spans}
traceBundler.Add(bundlerPayload, len(bundlerPayload.spans))
```

Reply:
Aha, nice reduction! I actually was going to write a variant of that but it was
too concise to not shock my future self and shout out the protocol.
I'll update that shortly though, thanks!

Reply:
Updated, thank you and PTAL!